.github/ISSUE_TEMPLATE/textual_bug.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Textual Bug
+about: Use this template to report grammar glitches, syntax errors, and typos.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+# Summary
+
+Replace this with a short summary (one sentence or short paragraph) of
+the issue you are describing.
+
+## Affected paths
+
+List the URL(s) where you found a problem. You can use URLs from [the
+rendered site](https://docs.cleura.cloud) or from [the sources on
+GitHub](https://github.com/citynetwork/docs), both is fine.
```

Comment:
Again, replace "both is fine" (grammatically incorrect) with "either is fine".

Reply:
Will do.